Testing/functional/tests/cypress/integration/e2e/user/organDonorCardQuickLink.js

```diff
@@ -0,0 +1,140 @@
+                .should("be.visible", "be.enabled")
+                .click();
+            cy.get(quickLinkRemoveButtonSelector).should("be.visible").click();
+        });
+
+        cy.log("Verifying organ donor quick link no longer exists");
+        cy.get(organDonorQuickLinkCardSelector).should("not.exist");
+
+        cy.log("Adding organ donor quick link");
+        cy.get(addQuickLinkButtonSelector)
+            .should("be.visible")
+            .should("be.enabled")
+            .click();
+        cy.get(organDonorAddQuickLinkCheckboxSelector)
+            .should("exist")
+            .check({ force: true });
+        cy.get(addQuickLinkSubmitButtonSelector)
+            .should("be.visible")
+            .should("be.enabled")
+            .click();
+
+        cy.log("Verifying add quick link button is disabled");
+        cy.get(addQuickLinkButtonSelector)
+            .should("be.visible")
+            .should("not.be.enabled");
+
+        cy.log("Verifying organ donor quick link card is present");
+        cy.get(organDonorQuickLinkCardSelector).should(
+            "be.visible",
+            "be.enabled"
+        );
+    });
+});
+
+describe("Disabling organ donor services", () => {
+    const testOrganDonorNotPresent = (cy) => {
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak,
+            homePath
+        );
+        cy.log("Verifying organ donor quicklink not present");
+        cy.get(organDonorQuickLinkCardSelector).should("not.exist");
+
+        cy.log("Verifying add organ donor checkbox not present");
+        cy.get(addQuickLinkButtonSelector)
+            .should("be.visible")
+            .should("be.enabled")
+            .click();
+        cy.get(organDonorAddQuickLinkCheckboxSelector).should("not.exist");
+    };
+
+    it("Should not show quick link and settings when services module is disabled", () => {
+        cy.configureSettings({
+            datasets: [
+                {
+                    name: "healthVisit",
+                    enabled: true,
+                },
+            ],
+            services: {
+                enabled: false,
+                services: [
+                    {
+                        name: "organDonorRegistration",
+                        enabled: true,
+                    },
+                ],
+            },
+        });
+        testOrganDonorNotPresent(cy);
+    });
+
+    it("Should not show quick link and settings when services.organDonor module is disabled", () => {
+        cy.configureSettings({
+            datasets: [
+                {
+                    name: "healthVisit",
+                    enabled: true,
+                },
+            ],
+            services: {
+                enabled: true,
+                services: [
+                    {
+                        name: "organDonorRegistration",
+                        enabled: false,
```

Comment:
First, configureSettings function in cypress/support.commands.js needs to be updated to support services-services-name/enabled.

Secondly, once configureSettings function has been updated, it is not required to set 'organDonorRegigistration' to false as that will be the default.